### ISSUE TITLE
Tagging a released version

### ISSUE THREAD

**Issue by dbu** [eko/FeedBundle#17]
Once dev-zend is merged into master, do you think it will become possible to tag a released version in the near future (like next month)?
we plan to tag stable releases of the cmf bundles in march, once symfony 2.2 became stable. the more dependencies provide tagged versions, the easier for us to model the dependencies.
